templates/packer/ansible/roles/common/files/monit_scripts/pozorbot_alert

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# Send alerts to pozorbot.
```

Comment:
Is this something we could trigger from cloudwatch rather than from the instance?